protobufs/livekit_internal.proto

```diff
@@ -81,7 +81,7 @@ message NodeStats {
   uint64 nack_total = 11;
   float bytes_in_per_sec = 12 [deprecated=true];
   float bytes_out_per_sec = 13 [deprecated=true];
-  float packets_in_per_sec = 14 [deprecated=true];
+  float packets_in_per_sec = 14;
```

Comment:
removing `deprecated` on this as it is used in capacity manager to check packet rate before using forwarding latency for load indication.